packages/flutter/lib/src/cupertino/dialog.dart

```diff
@@ -759,49 +755,302 @@ class CupertinoActionSheetAction extends StatelessWidget {
   }
 }
 
-class _CupertinoActionSheetCancelButton extends StatefulWidget {
-  const _CupertinoActionSheetCancelButton({
-    this.child,
+// This widget renders the background (pressed or not) of a button, and sends
+// whether the button is pressed to the parent.
```

Comment:
maybe
```
// This widget renders the background (pressed or not) of a button and calls `onPressStateChange`
// when user tap down or lift up on the button

Reply:
Thanks, I've changed it to
```dart
// Renders the background of a button (both the pressed background and the idle
// background) and reports its state to the parent with `onPressStateChange`.
```

(The exact definition of "the state" is explained in `onPressStateChange`.) What do you think of it?